lib/processes/selenium/index.js

```diff
@@ -43,29 +43,13 @@ var SELENIUM_TIMEOUT = 90000;
 
 var BIN_PATH = path.join(__dirname, '..', '..', '..', 'bin');
 var DEFAULT_JAR_PATH = path.join(BIN_PATH, 'selenium.jar');
-var DEFAULT_CHROME_PATH = path.join(BIN_PATH, 'chromedriver');
 
 var statAsync = Bluebird.promisify(fs.stat);
 var downloadSelenium =
   _.partial(Bluebird.promisify(SeleniumDownload.ensure), BIN_PATH);
 
 exports.name = 'selenium';
 
-function createSeleniumArguments(chromeDriverPath) {
-  var chromeArgs = [
-    '--disable-application-cache',
-    '--media-cache-size=1',
-    '--disk-cache-size=1',
```

Comment:
Are we removing the "disable X cache" logic on purpose?

Reply:
Nope, good catch. resurrected.